test/models/userProfileModelManager.test.js

```diff
@@ -0,0 +1,180 @@
+const { expect } = require('chai');
+const UserProfileModelManager = require('../../lib/modelManagers/userProfileModel');
+const UserModelManager = require('../../lib/modelManagers/usermodel');
+const { User, userprofile } = require('../../database/models');
+
+describe('User Profile Model Manager', async () => {
+  const newModels = {};
+  const userProfileMock = {
+    userId: 'testuserid',
+    avatar: 'https://testavarturl.com',
+    bio: 'Test bio',
+    username: 'user1',
+    country: 'Nigeria',
+    phonenumber: '7862',
+    gender: 'male',
+    emailnotification: false,
+    email: 'testuser@email.com',
+    password: 'pass',
+    firstname: 'firstname',
+    lastname: 'lastname'
+  };
+  describe('Create user profile', () => {
+    before(async () => {
+      const newUser = await UserModelManager.create(
+        userProfileMock.email,
+        userProfileMock.password,
+        userProfileMock.firstname,
+        userProfileMock.lastname
+      );
+      userProfileMock.userId = newUser.id;
+      const newUserProfile = await UserProfileModelManager.createUserProfile(
```

Comment:
move this ` const newUserProfile = await UserProfileModelManager.createUserProfile` into the `it block`